deployer/deployer-services/src/main/java/io/hyscale/deployer/services/handler/PodParentHandler.java

```diff
@@ -108,5 +120,55 @@ protected Object prepareScalePatch(ScaleOperation scaleOp, int scaleValue, int c
                 .build();
         return K8sResourcePatchUtil.getJsonPatch(exisiting, scale, V1Scale.class);
     }
+    
+    public PodParent getPodParent(ApiClient apiClient, String selector, boolean label, String namespace) throws HyscaleException {
+        List<PodParent> podParentList = getPodParentsList(apiClient, selector, label, namespace);
+        if (podParentList != null && !podParentList.isEmpty()) {
+            return podParentList.get(0);
+        }
+        return null;
+    }
+    
+    public List<PodParent> getPodParentsList(ApiClient apiClient, String selector, boolean label, String namespace)
```

Comment:
Use case of this method ?

Reply:
To get Deployment status using podParents

Reply:
Then it is either by Service or by App right ? Can you convey the same in the method name